src/ng/compile.js

```diff
@@ -1587,6 +1587,90 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     return cssClassDirectivesEnabledConfig;
   };
 
+
+  /**
+   * The security context of DOM Properties.
+   * @private
+   */
+  var PROP_CONTEXTS = Object.create(null);
+
+  /**
+   * @ngdoc method
+   * @name $compileProvider#addPropertyContext
+   * @description
+   *
+   * Defines the security context for HTML properties bound by ng-prop-*
+   * 
+   * @param {string} elementName the element name or '*' to match any element
+   * @param {string} propertyName the property name
+   * @param {string} ctx the context type
+   */
+  this.addPropertyContext = function(elementName, propertyName, ctx) {
+    var key = (elementName.toLowerCase() + '|' + propertyName.toLowerCase());
```

Comment:
I think Angular may have done that for a reason?

https://github.com/angular/angular/blob/6.0.6/packages/compiler/src/schema/dom_security_schema.ts#L29

Reply:
Interesting. OK then :smiley:
(The only "native" properties that are affected are `formAction`, `innerHTML` and `outerHTML`. The last two are weird as custom element property names anyway.)

Reply:
So in the [commit for that line of code](https://github.com/angular/angular/commit/040b101842b2bdb637caff91a2c547a4317e5100):

> Comparison of (potentially) dangerous property names is done case insensitive,
to avoid issues like formAction vs formaction.

I guess they are saying that in "some" browsers, "some" properties may be assignable using different casing?